src/mono/mono/mini/intrinsics.c

```diff
@@ -4,6 +4,7 @@
 
 #include <config.h>
 #include <mono/utils/mono-compiler.h>
```

Comment:
```suggestion
#include <glib.h>
#include <mono/utils/mono-compiler.h>
```

That should fix the C++ build problems in https://github.com/mono/mono/pull/20450

Reply:
Thanks!